lib/purl_fetcher/client/legacy_publish.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+module PurlFetcher
+  class Client
+    # Publish (metadata-only). This will be replaced with a single publish operation
+    class LegacyPublish
+      # @param [#to_json] cocina the Cocina data object
+      def self.publish(cocina:)
```

Comment:
Can you specify which flavor of cocina this is expecting?

Reply:
@justinlittman  do you mean "public" cocina?

Reply:
What are the classes that the parameter is expecting? There are many different cocina-related classes, e.g., Cocina::Models::DRO, Cocina::Models::DROWithMetadata, etc.).

Reply:
Here we just care that it responds to to_json and externalIdentifier.

Reply:
That's fine, but it doesn't help someone who is trying to use this client know how to use it correctly to send the right thing to Purl fetcher. The developer shouldn't be left to guess or understand how Purl fetcher works, especially since Purl fetcher has no openapi spec.

Reply:
I don't know what to say then.  The same thing we always sent?

Reply:
From the MetadataTransferService looks like it should be:
```
# @param [Cocina::Models::DRO,Cocina::Models::Collection] cocina
```